apps/rush-lib/src/examples/Legacy_CustomCommandLine.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Should you add a "$schema"?

It's okay to use a URL that is a 404 now, but will work by the time we publish